styles.css

```diff
@@ -0,0 +1,60 @@
+
+/* ......... the header section.......... */
+header {
+  width: 375px;
+  height: 48px;
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-around;
+  color: #fff;
+  padding-top: 44px;
+}
+
+.name {
+  font-family: 'Roboto', sans-serif;
+  line-height: 24px;
+}
+
+/* ............headline section............. */
+
+.headline {
+  background: #1c1a19;
+  color: #fff;
+  padding-bottom: 80px;
+  padding-top: 80px;
+  padding-left: 20px;
```

Comment:
- [ ] There is a white space at the bottom of the web, you can fix it by adding to the headline class a `height: 100vh` in this way you can set the height of that element to be as height as the view device is. This will make your web responsive on the height part 💯 

![image](https://user-images.githubusercontent.com/93630700/205456572-38db506d-90b3-44ae-95d5-194b97e9e293.png)